packages/create/README.md

```diff
@@ -148,6 +149,15 @@ Calling `sap-ux change inbound` allows replacing the Inbound FLP configurations
 sap-ux change inbound [path]
 ```
 
+## sap-ux convert
+Calling `sap-ux convert` allows converting an app to a new feature.
+
+### preview
+Calling `sap-ux convert preview-config` in the root folder of an app will convert the respective app to the preview with virtual files. It will use the configuration from the scripts in `package.json` to adjust the UI5 configuration yaml files accordingly. No longer needed js/ts sources will be deleted and the html files previously used for the preview will be renamed to *_old.html.
```

Comment:
```suggestion
Executing `sap-ux convert preview-config` in the root folder of an app will convert the respective app to the preview with virtual files. It will use the configuration from the scripts in the `package.json` file to adjust the UI5 configuration YAML files accordingly. The obsolete JS and TS sources will be deleted and the HTML files previously used for the preview will be renamed to `*_old.html`.
```

Reply:
Done in fe3e4ffa7a57333062a7629ce48b4f040708751f